### ISSUE TITLE
Please add support for gnome 43

### ISSUE THREAD

**Issue by Antonious-Awad** [Fausto-Korpsvart/Rose-Pine-GTK-Theme#4]

**Comment by Fausto-Korpsvart** [comment]
> please add compatibility for gnome 43

I will do it, I am slowly updating all the themes, it will surely arrive next week for Rosé Pine.
Thanks for your support.

**Comment by Antonious-Awad** [comment]
Thank you, It's been over a week and I just wanted to check if there's any new updates ^^

**Comment by Fausto-Korpsvart** [comment]
> Thank you, It's been over a week and I just wanted to check if there's any new updates ^^

Yes, I know, and I'm sorry, I've had some inconveniences and I haven't been able to update all the themes in the time I wanted to, but this week I hope to continue with all the other themes that are still to be updated.

Thanks for your support and for waiting.

**Comment by daVimci** [comment]
Hi, I just wonder that whether you will update the rose-pine GTK theme recently?  I'm really looking forward to it.

**Comment by Fausto-Korpsvart** [comment]
Hello, seriously I apologize because I know I have left for a moment the maintenance of the themes, it has been because of my work, but I have already finished the first theme `Rosé Pine Main`, I hope to upload it tomorrow, as I am missing the `Legacy Buttons` versions to be able to update `Rosé Pine Main` first, and then I will continue with the others.

This is how it looks at the moment.
![RosePine](https://user-images.githubusercontent.com/47998118/211066678-93db7c80-be44-43be-bdc5-5599454b0803.png)

Thank you for your support and patience.

**Comment by Fausto-Korpsvart** [comment]
Finally, the first update with support for Gnome 43, 
this is the Rosé Pine Main theme, I will update the remaining themes later this week.

Sorry for the delay, and thanks for your patience.
